test/envoye2e/stats_plugin/stats_test.go

```diff
@@ -365,3 +365,37 @@ func TestAttributeGen(t *testing.T) {
 		})
 	}
 }
+
+func TestStatsParserRegression(t *testing.T) {
+	// This is a regression test for https://github.com/envoyproxy/envoy-wasm/issues/497
+	params := driver.NewTestParams(t, map[string]string{
+		"StatsFilterCode":         "inline_string: \"envoy.wasm.stats\"",
+		"WasmRuntime":             "envoy.wasm.runtime.null",
```

Comment:
This happens even with NullVm?

Reply:
Yes. It's an issue with the host, not the module.